projects/ngx-speculoos/src/lib/matchers.ts

```diff
@@ -60,6 +60,38 @@ const speculoosMatchers: jasmine.CustomMatcherFactories = {
         return assert(true, el, expected);
       }
     };
+  },
+
+  /**
+   * Checks that the receiver is a TestElement wrapping a DOM element and contains the given textContent
+   */
+  toContainText: (util: jasmine.MatchersUtil, customEqualityTesters: Array<jasmine.CustomEqualityTester>): jasmine.CustomMatcher => {
+    const assert = (isNegative: boolean, el: any, expected: string) => {
+      if (!el) {
+        return { pass: false, message: `Expected to check textContent '${expected}' on element, but element was falsy` };
+      }
+      if (!(el instanceof TestElement)) {
+        return { pass: false, message: `Expected to check textContent '${expected}' on element, but element was not a TestElement` };
+      }
+      const actual = el.textContent;
+      if (!actual) {
+        return {
+          pass: isNegative,
+          message: `Expected element to ${isNegative ? 'not ' : ''}contain textContent '${expected}', but had no textContent`
+        };
+      }
+      const pass = actual.indexOf(expected) !== -1;
```

Comment:
actual.includes(expected)?

Reply:
This is es2016, so I had to change the tsconfig.json to add `es2016` if I wanted to use and I didn't know if that was good or bad...

Reply:
I don't measure all the implications, so leave it like this.